examples/controllers/Docker/init.sql

```diff
@@ -1,2 +1,2 @@
-CREATE DATABASE IF NOT EXISTS controllers;
```

Comment:
I have a PR for fixing these: https://github.com/Flexiana/framework/pull/44

Reply:
didn't see, sorry :(

Reply:
No problem, you are a good pioneer. ;)